### DISCUSSION TITLE
Is this still in active development

### DISCUSSION THREAD

**Discussion by JamesBotterill** [AR-js-org/AR.js#246] [General]
There's been little in the way of new releases in many many months and those who were active have either moved to their own thing or no longer active. 

Should we consider this as no longer being actively maintained?

**Comment by kalwalt** [comment]
You are right @JamesBotterill , there aren't new release by a long time... I am more involved in our github organization, but i would really continue with AR.js. For this reason would be great if the ES6 feature see PR #116 will become a reality. I am helping on this side @MikiDi  that did a great job, It is not finished but i hope that it will be merged at least in the dev branch soon and that a new NPM package will be available. I think this a great step for AR.js that we have to accomplish. :slightly_smiling_face:

**Comment by nickw1** [comment]
@JamesBotterill to add to @kalwalt's comment, I am planning to investigate the use of ground-plane detection to improve the realism of location-based AR at least, with an aim of placing 3D models realistically on the ground. I have been given some pointers on how to approach this by @ThorstenBux. However, due to time constraints I can only do this in the summer months, June to September or thereabouts.

**Comment by JamesBotterill** [comment]
@kalwalt @nickw1 That's great to hear, and yes ES6 would be great.

**Comment by kalwalt** [comment]
Moving this issue to discussions... :smile:
